contrib/test.bzl

```diff
@@ -109,7 +112,7 @@ _container_test = rule(
 def container_test(name, image, configs, driver=None, verbose=None):
     """A macro to predictably rename the image under test before threading
     it to the container test rule."""
-    intermediate_image_name = "%s:intermediate" % image.replace(':', '')
+    intermediate_image_name = "%s:intermediate" % re.sub(image_regex, '', image)
```

Comment:
This is not the Python you are looking for.  I don't think `re` is available...

Reply:
Ah whoops, forgot to import, but also forgot standard Python libs aren't available :( anything like `re` available?